.gitignore

```diff
@@ -26,3 +24,17 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 debug.log
+
+#amplify
+amplify/
+amplify/\#current-cloud-backend
+amplify/.config/local-*
+amplify/backend/amplify-meta.json
+amplify/backend/awscloudformation
+build/
+dist/
+node_modules/
+aws-exports.js
+aws_exports.js
+src/aws-exports.js
+awsconfiguration.json
```

Comment:
was this all added in by default after installing `aws-amplify`

Reply:
Yes, with the exception of the underscored aws exports because I think everyone has that in their repos right now